includes/discovery/functions.inc.php

```diff
@@ -916,12 +916,22 @@ function ignore_storage($os, $descr)
  * @param $value
  * @param $data
  * @param $group
+ * @param null $index
+ * @param array $pre_cache
  * @return bool
  */
-function can_skip_sensor($value, $data, $group)
+function can_skip_sensor($value, $data, $group, $index = null, $pre_cache = array())
```

Comment:
Don't pass the index, just pass the data for this one sensor (as in call the function with `$pre_cache[$index]`)

Reply:
Easy enough to do. Will update if we agree this is the way forward.

Reply:
Updated.

Reply:
Actually updated :/